classes/migration/upgrade/v3_4_0/I9136_MigrateUniqueSiteId.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/v3_4_0/I9136_MigrateUniqueSiteId.php
+ *
+ * Copyright (c) 2014-2023 Simon Fraser University
+ * Copyright (c) 2000-2023 John Willinsky
```

Comment:
Looks like we're using another pattern for new files: `Copyright (c) 2023 Simon Fraser University`